docs/apu2_vboot.md

```diff
@@ -0,0 +1,246 @@
+Vboot measured boot on apu2
+===========================
+
+This document describes a procedure of building a coreboot image for apu2 with
+vboot support and measured boot utilizing a TPM module. Procedure will work
+only on apu2 versions v4.9.0.6 and newer.
+
+## Building coreboot image
+
+1. Clone the [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
+2. Pull or [build](https://github.com/pcengines/pce-fw-builder#building-docker-image)
+  docker container:
+
+  ```
+  docker pull pcengines/pce-fw-builder
+  ```
+
+3. Build v4.9.0.6 image:
+
+  ```
+  ./build.sh release v4.9.0.6 apu2
+  ```
+
+4. Invoke distclean:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 distclean
+  ```
+
+5. Copy the vboot miniconfig:
+
+  ```
+  cp $PWD/release/coreboot/configs.pcengines_apu2_vboot $PWD/release/coreboot/.config
```

Comment:
@miczyg1 `./build.sh dev-build $PWD/release/coreboot apu2_vboot` should work IIRC

Reply:
Convention for naming was different also IIRC something like `configs.pcengines_apu2.debug`